src/components/UserProfile/UserProfileAccount/ProvidersLinkedCount/index.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { withStyles } from "@material-ui/styles";
+import InfoIcon from "@material-ui/icons/Info";
+import Typography from "@material-ui/core/Typography";
+
+import { useStyles } from "./styles";
+
+const ProvidersLinkedCount = ({ classes, providerCount = 0 }) => {
```

Comment:
we can use DefaultProps for providerCount instead

Reply:
Addressed in [commit](https://github.com/singnet/snet-dapp/pull/415/commits/86b2e60a4b25b4419233e1efc7b809cee8fccf40)